benchmarks/graphics_pipeline/main.cpp

```diff
@@ -379,8 +404,13 @@ void ProjApp::Setup()
         const uint32_t sphereVertexCount = mesh.GetCountPositions();
         const uint32_t sphereTriCount    = mesh.GetCountTriangles();
 
-        Geometry geo;
-        PPX_CHECKED_CALL(Geometry::Create(GeometryOptions::InterleavedU32().AddTexCoord().AddNormal().AddTangent(), &geo));
+        Geometry        lowPrecision;
+        GeometryOptions options;
+        options.IndexTypeU32().AddPosition(grfx::FORMAT_R16G16B16_FLOAT).AddTexCoord(grfx::FORMAT_R16G16_FLOAT).AddNormal(grfx::FORMAT_R16G16B16_FLOAT).AddTangent(grfx::FORMAT_R16G16B16A16_FLOAT);
```

Comment:
i think for normal and binormal, FORMAT_R8G8B8A8_SNORM should be good enough

Reply:
Updated.